src/services/auth.js

```diff
@@ -371,6 +372,45 @@ class UserService {
       user: user,
     };
   }
+
+  async changePassword(req) {
+    const { email, oldPassword, newPassword, confirmNewPassword } = req.body;
+
+    try {
+      const user = await User.findOne({ email: email });
```

Comment:
This will then be a findbyId using the userId from token

Reply:
at the moment the token does not contain this info
we will have to still use this email